assets/package.json

```diff
@@ -8,9 +8,11 @@
     "build:dev": "webpack --mode development"
   },
   "dependencies": {
+    "bootstrap": "^4.6.0",
+    "core-js": "^3.9.1",
+    "regenerator-runtime": "^0.13.7",
```

Comment:
Prefer to use exact versions for NPM packages to ensure stability:

```suggestion
    "regenerator-runtime": "0.13.7",
```

Reply:
I updated the versions.